lib/screens/StartupScreen.dart

```diff
@@ -840,19 +635,490 @@ class _StartPageState extends State<StartPage> {
+                                  enabledBorder: const OutlineInputBorder(
+                                      borderSide:
+                                      BorderSide(color: Colors.red)),
+                                  suffixIcon: const Icon(Icons.clear,
+                                      color: Colors.red),
+                                  hintText: 'Enter Last Name')
+                                  : globalDecoration.copyWith(hintText: 'Enter Last Name'),
+                              onChanged: (lastName) {
+                                if (lastName.isEmpty) {
+                                  setState(() => unfilledLastName = true);
+                                }
+                                else {
+                                  unfilledLastName = false;
+                                }
+                              },
+                              textInputAction: TextInputAction.next,
+                            ))
+                      ],
+                    ),
+                  ),
+                  Container(
+                    width: 210,
+                    padding: const EdgeInsets.only(top: 5),
+                    child: const Text(
+                      'Email',
+                      style: TextStyle(
+                          fontSize: 12,
+                          color: Colors.white,
+                          fontFamily: 'EagleLake'),
+                      textAlign: TextAlign.left,
+                    ),
+                  ),
+                  SizedBox(
+                    width: MediaQuery.of(context).size.width,
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        SizedBox(
+                          width: 210,
+                          height: 40,
+                          child: TextField(
+                            controller: _email,
+                            maxLines: 1,
+                            obscureText: false,
+                            decoration: unfilledEmail
+                                ? globalDecoration.copyWith(
+                                enabledBorder: const OutlineInputBorder(
+                                    borderSide:
+                                    BorderSide(color: Colors.red)),
+                                suffixIcon: const Icon(Icons.clear,
+                                    color: Colors.red),
+                                hintText: 'Enter Email')
+                                : globalDecoration.copyWith(hintText: 'Enter Email'),
+                            onChanged: (email) {
+                              if (!isEmail(email)) {
+                                errorMessage = 'Email must be in valid form';
+                                setState(() => unfilledEmail = true);
+                              }
+                              else {
```

Comment:
These should be on the same line `} else {`. Ditto elsewhere in places where you have try catch blocks etc. Should be able to use a formatter to fix